content/en/security/code_security/static_analysis/setup/_index.md

```diff
@@ -24,9 +24,9 @@ To set up Datadog SAST in-app, navigate to [**Security** > **Code Security**][1]
 ### Scan with Datadog-hosted scanning
 
 You can run Datadog Static Code Analysis (SAST) scans directly on Datadog infrastructure. Supported repository types include:
-- [GitHub](/security/code_security/static_analysis/setup/?tab=github) (excluding repositories that use [Git Large File Storage][17])
-- [GitLab.com and GitLab Self-Managed](https://docs.datadoghq.com/security/code_security/static_analysis/setup/?tab=gitlab)
-- [Azure DevOps](https://docs.datadoghq.com/security/code_security/static_analysis/setup/?tab=azuredevops)
+- [GitHub](/security/code_security/static_analysis/setup/?tab=github#select-your-source-code-management-provider) (excluding repositories that use [Git Large File Storage][17])
```

Comment:
these should be relative links, with the paths at the bottom of the page. We only link like this ^ inside of a tab.
I'll add these in this branch.